### PR TITLE
update(Update game entities): Updated Attackers and Defenders

### PR DESCRIPTION
**Pull request by vvivek1801**
### Updated Attackers and Defenders

- Updated the attacker and defender values
- Added aerial attribute

### CODE REVIEW

**Comment by Ram-20062003** [comment]
why did u delete pre-commit.sh?.

File: server/src/main/kotlin/delta/codecharacter/server/params/game_entities/Attacker.kt [hunk 1]
```diff
@@ -9,4 +9,5 @@ data class Attacker(
     @field:JsonProperty("attack_power", required = true) val attackPower: Int,
     @field:JsonProperty("speed", required = true) val speed: Int,
     @field:JsonProperty("price", required = true) val price: Int,
+    @field:JsonProperty("is_aerial", required = true) val type: Boolean,
```
Comment (Ram-20062003) [inline server/src/main/kotlin/delta/codecharacter/server/params/game_entities/Attacker.kt]:
```suggestion
    @field:JsonProperty("is_aerial", required = true) val attackerType: Boolean,
```

File: server/src/main/kotlin/delta/codecharacter/server/params/game_entities/Defender.kt [hunk 2]
```diff
@@ -8,4 +8,5 @@ data class Defender(
     @field:JsonProperty("range", required = true) val range: Int,
     @field:JsonProperty("attack_power", required = true) val attackPower: Int,
     @field:JsonProperty("price", required = true) val price: Int,
+    @field:JsonProperty("is_aerial", required = true) val type: Boolean,
```
Comment (Ram-20062003) [inline server/src/main/kotlin/delta/codecharacter/server/params/game_entities/Defender.kt]:
```suggestion
    @field:JsonProperty("is_aerial", required = true) val defenderType: Boolean,
```

**Review by Ram-20062003 — CHANGES_REQUESTED** [review]
## Changes Requested
- Fix the parameter namings and change it wherever it is being used

**Review by Ram-20062003 — APPROVED** [review]

**Review by Ashwin125 — APPROVED** [review]

**Pull request merged**